ui/Screen/DesignSystemGuide.svelte

```diff
@@ -636,6 +674,41 @@
     </Swatch>
   </Section>
 
+  <Section
```

Comment:
So far we've grouped all similar components under one section (and use `<Section>` tags to divide components into subgroups). Should we perhaps also do it for transactions for consistency's sake?

Reply:
It is grouped under the headline `Components`. Do you mean to have an additional `Transaction` group outside of `Components`?

Reply:
Yeah, something like this:
```
Components
  Avatars
  ...
  Transactions
    <Swatch><TransactionRow></Swatch>
    <Swatch><Accordion></Swatch>
    <Swatch><TransactionStatusbar></Swatch>
```